zips/zip-0228.rst

```diff
@@ -292,6 +311,21 @@ To be able to spend a subset of the Actions in a Swap Order, the matcher would h
 Given that :math:`\mathsf{bsk}` is a modular addition of random field elements, extracting specific values without additional context isn't possible. 
 Hence, :math:`\mathsf{bsk}` “binds together" the commitments of all Actions in the Order, preventing selective extraction of specific Actions in a Swap Order by a malicious matcher.
 
+Rationale for Fees
+``````````````````
+
+The proposed changes to the transaction structure and the consensus checks also require a change in the fees paid to miners.
+The use of Action Groups adds information to the transaction object, and adds more bytes to the blockchain state. 
+This needs to be paid for by network users and will result in an increase in the expected fee, though the logic is the same as in the ZSA Protocol.
+
+The Matcher Fees incentivize the Matcher to search through the unmatched Swap Orders and create Swap Bundles out of matching Orders.
+It also compensates the Matcher for the Miner Fees that will arise when the constructed Swap Bundle is included in a transaction.
+For the matching to be economically viable, the fees captured by the Matcher to create the Swap Bundle must at least be higher than the fees needed to settle the bundle on the chain.
```

Comment:
Reconsider terminology.

Reply:
I think it is better to stick to Miner and Matcher fees, adding an additional Total fee term only complicates it further